homeassistant/components/ping/binary_sensor.py

```diff
@@ -56,6 +56,7 @@
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_UNIQUE_ID): cv.string,
```

Comment:
We should not add a unique ID to this integration, instead this should be moved to an configuration flow.